package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -458,19 +458,53 @@ class DumpReader(base.ReaderBase):
     "unwrapped" (xu,yu,zu) and "scaled_unwrapped" (xsu,ysu,zsu) coordinate
     conventions (see https://docs.lammps.org/dump.html for more details).
     If `lammps_coordinate_convention='auto'` (default),
-    one will be guessed. Guessing checks whether the coordinates fit each convention in the order "unscaled",
-    "scaled", "unwrapped", "scaled_unwrapped" and whichever set of coordinates
-    is detected first will be used. If coordinates are given in the scaled
-    coordinate convention (xs,ys,zs) or scaled unwrapped coordinate convention
-    (xsu,ysu,zsu) they will automatically be converted from their
-    scaled/fractional representation to their real values.
+    one will be guessed. Guessing checks whether the coordinates fit each
+    convention in the order "unscaled", "scaled", "unwrapped",
+    "scaled_unwrapped" and whichever set of coordinates is detected first will
+    be used. If coordinates are given in the scaled coordinate convention
+    (xs,ys,zs) or scaled unwrapped coordinate convention (xsu,ysu,zsu) they
+    will automatically be converted from their scaled/fractional representation
+    to their real values.
 
     Supports both orthogonal and triclinic simulation box dimensions (for more
     details see https://docs.lammps.org/Howto_triclinic.html). In either case,
     MDAnalysis will always use ``(*A*, *B*, *C*, *alpha*, *beta*, *gamma*)``
     to represent the unit cell. Lengths *A*, *B*, *C* are in the MDAnalysis
     length unit (Å), and angles are in degrees.
 
+    By using the keyword `additional_columns`, you can specify arbitrary data
+    to be read alongside the coordinates. If specified, the keyword expects a
+    list of the names of the columns that you want to have read. The results
+    of the parsing are saved to the time step `data` dictionary alongside the
```

Comment:
Instead of `data`, use reST role
```reST
of the parsing are saved to the time step :attr:`Timestep.data` dictionary alongside the
```
Check that the generated link resolves properly in the docs or do something like 
```
:attr:`Timestep.data <MDAnalysis.coordinates.base.Timestep.data>`
```